### DISCUSSION TITLE
大家的需求非常多样，是否考虑将prompt自定义

### DISCUSSION THREAD

**Discussion by lingtongwang** [nextai-translator/nextai-translator#554] [Ideas]
各项功能大家都有不同的个性需求，是否考虑在现有功能基础上，开放用户自定义prompt的功能，用户可以根据自己需求，在固定功能基础上增加prompt；或者是可以将玩家需求的功能汇总，编成功能可增加的可选项供用户选择。

**Comment by FunkyTo9** [comment]
真希望开发者能增加大文本、txt文件翻译功能，因为复制太多实在是很累人的。

**Comment by yetone** [comment]
已经在计划中了
